components/datetime/src/date.rs

```diff
@@ -163,6 +165,30 @@ fn week_of_year<T: DateInput>(
     Ok(WeekOfYear(u32::from(week.week)))
 }
 
+/// Returns the week of month according to a calendar with min_week_days = 1.
+///
+/// This is different from what the UTS35 spec describes but the latter is
```

Comment:
Is there a link to the section of the spec you are referring to?

Reply:
Yes, I've added a link. For week-of-month it boils down to "Values are similarly calculated for the Week of Month.".